docs/en/api/metrics-query-expression.md

```diff
@@ -296,6 +296,24 @@ view_as_seq(not_existing, service_cpm)
 #### Result Type
 The result type is determined by the type of selected not-null metric expression.
 
+### IsPresent Operation
+IsPresent operation represents that in a list of metrics, if any expression has a value, it would return `1` in the result; otherwise, it would return `0`.
+
+Expression:
+```text
+is_present([<expression_1>, <expression_2>, ...])
```

Comment:
Are the expressions SINGLE VALUE or TIMER SERIES VALUE? Or either?

Reply:
Both are supported.